src/subscription_manager/cli_command/abstract_syspurpose.py

```diff
@@ -187,7 +187,9 @@ def _are_provided_values_valid(self, values):
         valid_fields = self._get_valid_fields()
         if self.attr in valid_fields:
             for value in values:
-                if value not in valid_fields[self.attr]:
+                if any([x.casefold() == value.casefold() for x in valid_fields[self.attr]]):
```

Comment:
there are two options in this case:
- simply negate this `any()`, so it becomes: if the wanted `value` is not found with the valid fields
- turn the `any()` into `all()` and the `==` into `!=`, so it becomes: if the wanted `value` is different from all the valid fields

Reply:
went the second route